Add files via upload

Neschadin opened the pull request.

Neschadin committed:
Add files via upload

Neschadin committed:
Update gentx-e4ed0aac1a963da9028393494fa49cf5d1303e9b.json

Neschadin committed:
Merge branch 'haqq-network:master' into master

icodragon commented:
Hello! Please regenerate gentx (in the guide we set a new amount value) and do not change the amount manually, trust this CLI

Neschadin committed:
Update and rename gentx/gentx-e4ed0aac1a963da9028393494fa49cf5d1303e9b.json to StakeBroker.json

changed amount,
also changed the description and location (IP) of the validator

Neschadin reviewed (commented):
**regenerate gentx:**
changed amount,
also changed the description and location (IP) of the validator

Neschadin commented:
> Hello! Please regenerate gentx (in the guide we set a new amount value) and do not change the amount manually, trust this CLI

Done, sir!

Neschadin committed:
Rename StakeBroker.json to gentx/StakeBroker.json

move to 
gentx/
